Makefile

```
@@ -0,0 +1,10 @@
+.PHONY: build test serve
+
+build:
+	jb build specification
```

Might as well add a pdf target. There should be a way to make this more generic, such that `make $BUILDER` translates to `jb build --builder $BUILDER specification`, but I don't have the energy to look up Makefile syntax right now.

```suggestion
html:
	jb build specification

pdf:
    jb build --builder latex specification
```